flatten_json_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 )
 
-func bequal(a []byte, b []byte) bool {
+func equal(a []byte, b []byte) bool {
```

Comment:
Instead of using this method, we can use golang `bytes.Equal` (https://pkg.go.dev/bytes#Equal)

If you don't want to change in this PR, I can open another one.

Reply:
Yes please, let's not mix up CI PRs with code PRs.  Thanks in advance.